### DISCUSSION TITLE
How build Windows executable

### DISCUSSION THREAD

**Discussion by PABLOCORRO** [LibreDWG/libredwg#340] [General]
Hello everybody
I would like help, could you guide me on how to build the executable for dwg2dxf.c. I know only the basics of C ++ but no more.
I have installed MSYS64 successfully and C:\msys64\mingw64\bin contains tools. I have run the commands: automake, aclocal, configure; but I have not been able **to generate the .exe for dwg2dxf.c**
What are the commands that I should execute?  and in what order?

**Comment by rurban** [comment]
See the recipes in .appveyor.yml and .github/workflow/main.yml
